libs/components/atoms/src/lib/empty-biomarker/empty-biomarker.component.ts

```diff
@@ -0,0 +1,16 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatButtonModule } from '@angular/material/button';
+
+@Component({
+  selector: 'hra-empty-biomarker',
+  standalone: true,
+  imports: [CommonModule, MatButtonModule],
+  templateUrl: './empty-biomarker.component.html',
+  styleUrls: ['./empty-biomarker.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class EmptyBiomarkerComponent {
+  @Input()
+  buttonText = 'Collaborate with the HRA Team';
```

Comment:
- Move to same line as `@Input()`
- Rename to `collaborateText`
- Move default value to story. Use empty string as the default instead